### ISSUE TITLE
Blockquote example does not need figure/figcaption

### ISSUE THREAD

**Issue by aardrian** [mdn/interactive-examples#2563]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote

### What specific section or headline is this issue about?

Try it

### What information was incorrect, unhelpful, or incomplete?

The "Try it" section wraps the blockquote text in a figure with a figure caption, but does not explain why the figure is used nor explain the impact on screen reader announcements.

The code:
```
<figure>
    <blockquote cite="https://www.huxley.net/bnw/four.html">
        <p>Words can be like X-rays, if you use them properly—they’ll go through anything. You read and you’re pierced.</p>
    </blockquote>
    <figcaption>—Aldous Huxley, <cite>Brave New World</cite></figcaption>
</figure>
```

### What did you expect to see?

Either:
- no `<figure>` element, or
- an explanation why it is used in this section but not in the "Examples" section later in the page.

### Do you have any supporting links, references, or citations?

Fire up a screen reader and navigate it.

This is the output navigating with the virtual cursor in NVDA/Firefox:

`↓`
> em dash Aldous Huxley, Brave New World, figure, block quote, left quote Words can be like X dash rays, if you use them properly em dash they’ll go through anything. You read and you’re pierced. right quote

`↓`
> out of block quote, caption, em dash Aldous Huxley, Brave New World

The `<figure>` causes an announcement of the figure, which is fine but adds unnecessary noise. The `<figcaption>` causes announcement of the text it contains at the _start_ of the content, prior to announcing the presence of the figure or blockquote Then the contents of the `<figcaption>` are announced again as the user moves the virtual cursor through the DOM.

As a user, it is unclear why I am hearing this information twice (other than the code chosen). As a developer, there is no guidance on what this code will output.

This is the output navigating with the virtual cursor in JAWS/Chrome:

`↓`
> block quote cited from https://www.huxley.net/bnw/four.html

`↓`
> has details, left quote Words can be like X dash rays, if you use them properly em dash they’ll go through anything. You read and you’re pierced. right quote
block quote end

`↓`
> has details, em dash Aldous Huxley, Brave New World

Here, the "has details" announcement is a function of the `<figcaption>`.

### Do you have anything more you want to share?

I cannot edit this example in GitHub because it comes in from an embed with no immediately apparent documentation on where to find it:
`{{EmbedInteractiveExample("pages/tabbed/blockquote.html","tabbed-standard")}}`

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/element/blockquote`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/blockquote/index.md
* Last commit: https://github.com/mdn/content/commit/e04d8d2766c468f149445c0bf438d09f9b2d188c
* Document last modified: 2023-06-13T08:10:47.000Z

</details>

**Comment by myfonj** [comment]
Apparently added by https://github.com/mdn/interactive-examples/commit/25f7964c46f1e0bd62e2535a681e32e82c737a84 changed from footer in the blockquote.

**Comment by aardrian** [comment]
Ah, so I can edit https://github.com/mdn/interactive-examples/blob/main/live-examples/html-examples/text-content/blockquote.html.

I'll just go do that.

**Comment by NicolaSaunders** [comment]
Another example of wrapping a `blockquote` in a `figure` element exists at https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure#quotations.

Should this be removed?

**Comment by myfonj** [comment]
That one is [in content (repo) directly][1], and apparently it is there from the [start (migration)][2], so hard to find who authored it.  But as for OP snippet, maybe we could ask @mikoMK for pointers? Maybe there is some argument that could potentially overweight current NVDA/Firefox flaw (?)

My guess is it might have been taken directly from the HTML specs where this ("fancy") [`figure > blockquote + figcaption`][6] example is currently present as well.

I don't want to complicate it and I agree that MDN should rather promote _real-world best practices_ than ideal but actually flawed techniques, so if original ("fancy") code is flawed in any fairly recent user agent, it is a good thing to correct it ("dumb it down") for sake of broadest support.

But maybe (maybe?) if the original "fancy" structure is otherwise working and widely agreed upon **and** the problem in question is a problem of concrete product (NVDA/Firefox), it should be mentioned in docs and linked to relevant product bugreports? (Or we should both dumb it down **AND** mention reasoning why it diverges from the specs / ideal structure?)

Moreover, current docs for [`<footer>`] tells that semantically it complements **sectioning content** which is [defined][4] as:

> [`<article>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article), [`<aside>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside), [`<nav>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav), and [`<section>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section).

(It is basically quoted from the [whatwg DOM specs][5].)  There is no `<blockquote>` listed as the sectioning content, so as I understand it from specs perspective putting `<footer>` into `<blockquote>` is somewhat problematic. (Yet, again, real-world, not specs, is what really matters, so take it just as a remark.)

[1]: https://github.com/mdn/content/blob/6ad2974f7c4540f664bea27894de0cba47106556/files/en-us/web/html/element/figure/index.md?plain=1#L70-L78
[2]: https://github.com/mdn/content/blob/cbe151a06d6e5b4d1fbb46081bd16e69ef4c1630/files/en-us/web/html/element/figure/index.html#L110-L115
[3]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer
[4]: https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories#sectioning_content
[5]: https://html.spec.whatwg.org/multipage/dom.html#sectioning-content-2
[6]: https://html.spec.whatwg.org/multipage/grouping-content.html#the-blockquote-element:the-blockquote-element-5

**Comment by aardrian** [comment]
@NicolaSaunders 
> Should this be removed?

IMO no, since that shows assorted ways to use `<figcaption>` even if meh.

@myfonj 
> Maybe there is some argument that could potentially overweight current NVDA/Firefox flaw (?)

What NVDA/Firefox flaw?

> (It is basically quoted from the [whatwg DOM specs](https://html.spec.whatwg.org/multipage/dom.html#sectioning-content-2).) There is no `<blockquote>` listed as the sectioning content, so as I understand it from specs perspective putting `<footer>` into `<blockquote>` is somewhat problematic. (Yet, again, real-world, not specs, is what really matters, so take it just as a remark.)

Not problematic. Just not exposed as a landmark.

**Comment by myfonj** [comment]
> What NVDA/Firefox flaw?

The _"unnecessary noise"_ produced by reading the `figcaption` twice, as you described:

> `↓`
>> *em dash* **Aldous Huxley, Brave New World**, *figure, block quote, left quote* Words can be like X *dash* rays, if you use them properly *em dash* they’ll go through anything. You read and you’re pierced. *right quote*

> `↓`
>> *out of block quote, caption, em dash* **Aldous Huxley, Brave New World**

**Comment by aardrian** [comment]
> What NVDA/Firefox flaw?
> > The "unnecessary noise" produced by reading the figcaption twice, as you described:

Ah. I don't consider that a bug.

**Comment by estelle** [comment]
just some historical context: the example in the docs is b/c of the example in whatwg

**Comment by aardrian** [comment]
If it satisfies anyone's curiosity, I wrote about different announcements of 7 patterns across 6 screen readers (so far):
https://adrianroselli.com/2023/07/blockquotes-in-screen-readers.html

It includes the pattern I replaced, which was the most verbose by far across NVDA, JAWS, and macOS VoiceOver. Still not a bug, IMO, just unnecessary code.
